tests/api_tests/nodes/test_views.py

```diff
@@ -2447,8 +2447,9 @@ def test_return_public_node_pointers_logged_out(self):
         res_json = res.json['data']
         assert_equal(len(res_json), 1)
         assert_equal(res.status_code, 200)
-        assert_in(res_json[0]['attributes']['target_node_id'], self.public_pointer_project._id)
         assert_equal(res.content_type, 'application/vnd.api+json')
+        url = res.json['data'][0]['relationships']['target_node']['links']['related']['href']
+        assert_in('/{}nodes/{}/'.format(API_BASE, self.public_pointer_project._id), url)
```

Comment:
I'm asserting that the first is in the second.  For some reason, the url being returned here has localhost:80 instead of 8000 so I just comare the /v2/nodes/pgvjg.  Outside of the tests, the url is returned correctly localhost:8000/...

![screen shot 2015-10-14 at 6 46 54 pm](https://cloud.githubusercontent.com/assets/9755598/10500164/368b7976-72a4-11e5-8617-e8b44fa3f25f.png)